src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -86,55 +85,44 @@ public class SendReceiveTest : ClientWebSocketTestBase
         {
             using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
             {
-                var cts = new CancellationTokenSource(TimeOutMilliseconds);
+                var tasks = new Task[2];
```

Comment:
Yeah, but the exception should be thrown on the first outstanding `SendAsync` that occurs - additional invocations only increase the chance one of them fails (which is still potentially important, yes).  Problem was, the original data size, plus the way the test was written, hid the fact that _no exceptions were being thrown_.  I tried increasing the element count, but that didn't help; increasing the size did.